src/plugins/discord/sqliteMirror.js

```diff
@@ -497,4 +497,20 @@ export class SqliteMirror {
       .all({channel_id: channel, message_id: message})
       .map((e) => Model.refToEmoji(e.emoji));
   }
+
+  nthMessageIdFromTail(channel: Model.Snowflake, n: number) {
```

Comment:
Should define a return type. `?Model.Snowflake`

Reply:
updated